### DISCUSSION TITLE
DataTable: disable the `rowEditor` prop dynamically, as is done in `rowExpansion`

### DISCUSSION THREAD

**Discussion by datacampsis** [primefaces/community#756] [PrimeReact] [answered]
In rowExpansion, we can disable the prop `expander` if data has a specific value.
Example:
`const allowExpansion = (rowData: Product) => {
        return rowData.orders!.length > 0;
    };`
    
 In the `rowEditor`, there is no such option.
 
 Add the option to disable this prop with a specific condition. Like this:
 
 `const allowEditor = (rowData: Product) => {
        return rowData.orders!.length > 0;
    };`
    
    <Column 
         rowEditor={allowEditor}
    />

**Comment by melloware** [comment, marked as answer]
Fixed for next release: https://github.com/primefaces/primereact/issues/5640
